crates/livekit_client/src/lib.rs

```diff
@@ -3,42 +3,45 @@ use collections::HashMap;
 mod remote_video_track_view;
 pub use remote_video_track_view::{RemoteVideoTrackView, RemoteVideoTrackViewEvent};
 
-#[cfg(not(any(
-    test,
-    feature = "test-support",
-    all(target_os = "windows", target_env = "gnu"),
-    target_os = "freebsd"
-)))]
+// #[cfg(not(any(
+//     test,
+//     feature = "test-support",
```

Comment:
This should be undone

Reply:
Note to self, do not use the Github UI as diff viewer :)